examples/chip-tool-darwin/commands/tests/TestCommandBridge.h

```diff
@@ -104,31 +106,24 @@ class TestCommandBridge : public CHIPCommandBridge,
         NextTest();
     }
 
-    /////////// DelayCommands-like Interface /////////
-    void WaitForMs(unsigned int ms)
+    void UserPrompt(const char * _Nullable identity, const chip::app::Clusters::LogCommands::Commands::UserPrompt::Type & value)
     {
-        chip::app::Clusters::DelayCommands::Commands::WaitForMs::Type value;
-        value.ms = ms;
-        WaitForMs(nil, value);
+        NextTest();
     }
 
-    void WaitForMs(const char * _Nullable identity, const chip::app::Clusters::DelayCommands::Commands::WaitForMs::Type & value)
+    /////////// DelayCommands-like Interface /////////
```

Comment:
```suggestion
    /////////// DelayCommands Interface /////////
```
Since this is in fact now DelayCommands, right?